homeassistant/components/device_automation/__init__.py

```diff
@@ -19,9 +22,26 @@
     hass.components.websocket_api.async_register_command(
         websocket_device_automation_list_triggers
     )
+
+    component = hass.data[DOMAIN] = EntityComponent(
```

Comment:
The purpose of this is just to make `async_setup_entry` work, is there a way to do it without `EntityComponent`?